From 30a3435aa5b86089683fe6f3fd859c009651075f Mon Sep 17 00:00:00 2001
From: Marco Meyer-Conde <marco@tcu.ac.jp>
Date: Sat, 22 Feb 2025 19:54:44 +0900
Subject: [PATCH] stdlib

---
 src/FrameL.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/FrameL.h b/src/FrameL.h
index df9fdb3..19ba9cf 100644
--- a/src/FrameL.h
+++ b/src/FrameL.h
@@ -11,8 +11,7 @@
 #define FRPI              3.14159265358979
 #define FRTWOPI           6.2831853071795864769
 
-
-
+#include <stdlib.h>
 
 #ifdef __cplusplus
 extern "C" {
@@ -21,7 +20,6 @@ extern "C" {
 #include <time.h>
 #include <string.h>
 #include <stdio.h>
-#include <stdlib.h>
 #include <limits.h>
 #include "FrIO.h"
 
-- 
2.34.1

